packages/application-shell/src/components/project-switcher/project-switcher.spec.tsx

```diff
@@ -32,8 +26,7 @@ describe('rendering', () => {
 
   it('should search and select a project', async () => {
     render();
-    await screen.findByLabelText('Project switcher');
-    const input = screen.getByLabelText('Project switcher');
+    const input = await screen.findByLabelText('Projects menu');
```

Comment:
We were finding and element and then getting it but it seems more straightforward just to do it in one line.